Resources/Textures/DeltaV/Objects/Decoration/Misc/jackolanternunlit.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, modified by TJohnson.",
```

Unrelated commit, permalink to the file.
```suggestion
    "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432   modified by TJohnson.",
```